### PR TITLE
chore(weave): Fix call query on the Playground

### PR DESCRIPTION
**Pull request by jwlee64**
## Description

https://wandb.ai/shawn/programmerjs-swebenchcode-junk2/weave/calls/0193d638-3602-7316-ba5a-f242fe4a5408

this open in playground, didnt load the call because the call query fails with include_cost
temporary fix until I fix cost query

## Testing

How was this PR tested?

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx [hunk 1]
```diff
@@ -103,6 +115,16 @@ export const PlaygroundPageInner = (props: PlaygroundPageProps) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [call.loading]);
 
+  useEffect(() => {
+    if (!callWithCosts.loading && callWithCosts.result) {
+      if (callWithCosts.result.traceCall?.inputs) {
+        setPlaygroundStateFromTraceCall(callWithCosts.result.traceCall);
+      }
+    }
+    // Only set the call the first time the page loads, and we get the call
```
Comment (gtarpenning) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx:124]: wait does this mean the query fires multiple times?

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx [hunk 2]
```diff
@@ -52,6 +52,18 @@ export const PlaygroundPageInner = (props: PlaygroundPageProps) => {
...
+    useMemo(() => {
+      return props.callId
+        ? {
+            entity: props.entity,
+            project: props.project,
+            callId: props.callId,
+          }
+        : null;
+    }, [props.entity, props.project, props.callId])
+  );
```
Comment (gtarpenning) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx:64]: might be easier to reason about if we assign this to a variable

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx (hunk 1 above)
Comment (jwlee64) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx:124]: it does not, but if we add setPlaygroundStateFromTraceCall to the deps, this hook runs a bunch

Comment (gtarpenning) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx:124]: lets memo setPlaygroundStateFromTraceCall then :)

Comment (gtarpenning) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx:124]: if we don't want to do that upstream for some reason we can just make a local memo'd version in this scope

Comment (jwlee64) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx]: fixed

**Review by gtarpenning — APPROVED** [review]
We didn't need to memo setPlaygroundStateFromTraceCall?

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx (hunk 1 above)
Comment (jwlee64) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx]: it turns out it was memoized already 😝

**Pull request merged**